README.rst

```diff
@@ -1,6 +1,13 @@
 STARDIS Stellar Radiative transfer
 ----------------------------------
 
+Welcome to STARDIS!
+STARDIS is an LTE radiative transfer Python package that utilizes the TARDIS infrastructure 
+to generate synthetic stellar spectra. For more information about STARDIS, 
+including a quickstart notebook, physics walkthrough and installation instructions, go to our `documentation page <https://tardis-sn.github.io/stardis/>`_. 
+Or, try running the `quickstart notebook <docs/quickstart/quickstart.ipynb>`_ yourself!
+
+To learn more about TARDIS, including documentation and collaborator information, go `here <https://tardis-sn.github.io/tardis/>`_.
```

Comment:
Looks great. Just add a bit of whitespace after the final line of documentation. Currently the word License on line 11 ends up on the same line as the text from line 10. Add a newline character here or an extra line break to separate them.